site/content/docs/03-template-syntax.md

```diff
@@ -1014,6 +1014,16 @@ Like actions, transitions can have parameters.
 {/if}
 ```
 
+##### Transition options
+
+---
+
+Transitions also get passed an options object which provides data about the type of transition.
+
+Possible values in the options object are:
+
+* `direction` - one of `in`, `out`, or `both` depending on the type of transition
+
```

Comment:
[imo]

I think it would be better to put sample code in the `Custom transition functions` section and remove this section.
Because there may be people who don't understand the use case from only this explanation.

Reply:
Updated, lmk if that's more like what you were hoping for.